src/index.js

```diff
@@ -41,15 +39,24 @@ class NetlifyAPI {
   }
 
   get accessToken() {
-    return (get(this, 'defaultHeaders.Authorization') || '').replace('Bearer ', '') || null
+    const {
+      defaultHeaders: { Authorization },
+    } = this
+    if (typeof Authorization !== 'string' || !Authorization.startsWith('Bearer ')) {
```

Comment:
This logic is slightly different, but seems to capture the actual intent. @erezrokah What do you think?

Reply:
Yes, this looks much better